addons/bestja_project/models.py

```diff
@@ -239,6 +239,12 @@ class UserWithProjects(models.Model):
         inverse_name='manager'
     )
 
+    def __init__(self, pool, cr):
+        super(UserWithProjects, self).__init__(pool, cr)
+        cls = type(self)
+        cls.FIELDS_ACCESS_OWNER += ['projects', 'managed_projects']
+        cls.FIELDS_ACCESS_PRIVILEGED += ['projects', 'managed_projects']
```

Comment:
The same remark that was made earlier – we should not modify the class it instance constructor, at least without checking that the monkey patch has been already performed.